### PR TITLE
feat: implemented trade success ui for multi-hop trades

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Implements the trade success ui for multi hop trades.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

NA

## Risk

Low risk as no production-facing code is changed.

## Testing

Not required.

## Screenshots (if applicable)

Successful trade with details expanded - note the success ui shows the user BALANCE, not the amount that was received during the trade (cc @reallybeard)
image

Successful trade with details hidden
image

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5639** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5639?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5639?utm_source=stack-comment).

File: src/components/MultiHopTrade/MultiHopTrade.tsx [hunk 1]
```diff
@@ -85,15 +85,17 @@ export const MultiHopTrade = memo(
...
+        <Card {...cardProps}>
+          <CardBody px={0} py={0}>
+            <FormProvider {...methods}>
+              <MemoryRouter initialEntries={MultiHopEntries} initialIndex={0}>
+                <MultiHopRoutes />
+              </MemoryRouter>
+            </FormProvider>
+          </CardBody>
+        </Card>
+      </>
```
Comment (woodenfurniture) [inline src/components/MultiHopTrade/MultiHopTrade.tsx:98]: remove <>

**Review by 0xApotheosis — APPROVED** [review]
Happy with this guy as-is considering it's behind a flag.

File: src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    if (!lastHop) return ''
+
+    const manager = getChainAdapterManager()
+    const adapter = manager.get(lastHop.buyAsset.chainId)
+
+    if (!adapter) return ''
+
+    const cryptoAmountFormatted = toCrypto(totalCryptoHumanBalance, lastHop.buyAsset.symbol)
+    const chainName = adapter.getDisplayName()
+    return `You now have ${cryptoAmountFormatted} in your wallet on ${chainName}.`
```
Comment (gomesalexandre) [inline src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx:57]: Translation?

File: src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        </SlideTransition>
+      </CardBody>
+      <CardFooter flexDir='column' gap={2} px={4}>
+        <SlideTransition>
+          <Button mt={4} size='lg' width='full' onClick={handleBack} colorScheme='blue'>
+            Do another trade
+          </Button>
+          <HStack width='full' justifyContent='space-between' mt={4}>
+            <Button variant='link' onClick={onToggle} px={2}>
+              Show Details
```
Comment (gomesalexandre) [inline src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx:88]:
```suggestion
              translate('common.showDetails')
```

**Review by gomesalexandre — APPROVED** [review]
Conceptually happy with this guy minus missing translations, can be tackled all at once across multi-hop domain as a follow-up if you feel like it 🤌

**Pull request merged**